### PR TITLE
Use DB Browser for SQLite instead of Firefox

### LINKED ISSUE #199: SQLite Manager and new Firefox
Amy Hodge points out on [this thread](http://lists.software-carpentry.org/pipermail/discuss/2017-November/005585.html) that the new release of Firefox does not support the SQLite Manager plugin. Couple of things we should do:

* (short term): update the setup instructions letting instructors / learners know which version of Firefox they need to be able to run the plugin
* (long term): explore other options. In the discuss thread, @fmichonneau suggests [Falcon](https://plot.ly/database-connectors/) and [sqlitebrowser](https://github.com/sqlitebrowser/sqlitebrowser) as possibilities.

### PR DESCRIPTION
**Pull request by remram44**
Initial take at updating the lesson after SQLite Manager no longer works on Firefox.

Fixes #199

### CODE REVIEW

**Comment by remram44** [comment]
I also fixed the empty quote blocks on the setup page:

screenshot of empty quote blocks

File: setup.md [hunk 1]
```diff
@@ -5,24 +5,24 @@ permalink: /setup/
...
-available on FigShare, with a CC-BY license 
-available for reuse.
 
 
 > ## Software
+> SQL is a specialized programming language used with databases.  We
+> use a simple database manager called [SQLite](http://www.sqlite.org/)
+> in our lessons.
+> 
+> We will use the [DB Browser for SQLiter](http://sqlitebrowser.org/) program,
```
Comment (justinclift) [inline setup.md]: Typo here. "SQLiter" -> "SQLite" :wink:

Comment (remram44) [reply setup.md]: Thanks for catching that!

Comment (justinclift) [reply setup.md]: No worries. :smile:

**Comment by ErinBecker** [comment]
Thanks Rémi!

**Pull request merged**